app/src/models/enums/IReportFields.ts

```diff
@@ -0,0 +1,15 @@
+export enum IReportFields {
```

Comment:
Only interfaces need to start with an `I`, so this can just be called `ReportFields` :)

Reply:
Oh I see. I've been wondering what the 'I' stands for